src/witan/workspace/handler.clj

```diff
@@ -21,15 +22,33 @@
             :tags [{:name "api", :description "some apis"}]}}}
 
    (GET "/by-owner" req
-        :summary "Just a test"
-        :components [db]
-        :query-params [owner :- s/Uuid]
-        (do
-          (log/debug req)
-          (w/by-owner owner (params-vector req :fields) db)))
+     :summary "Queries the workspaces by owner"
+     :components [db]
+     :query-params [owner :- s/Uuid]
+     (do
+       (log/debug req)
+       (w/by-owner owner (params-vector req :fields) db)))
```

Comment:
how does this handle missing owners? I'm guessing buddy will protect the URL at the ring level? Can we be DDoS'd by people guessing owners if they have created a user id?